Documentation/Books/Manual/Programs/Arangodump/Examples.md

```diff
@@ -30,6 +30,10 @@ Here's an example of dumping data from a non-standard endpoint, using a dedicate
 
     arangodump --server.endpoint tcp://192.168.173.13:8531 --server.username backup --server.database mydb --output-directory "dump"
 
+In contrast to the above call `--server.database` must not be specified when duming all databases using `--all-databases true`:
```

Comment:
`true` is not needed for binary options, but probably better to be explicit so 👍